Mathlib/RingTheory/Jacobson.lean

```diff
@@ -453,8 +453,7 @@ theorem isJacobson_polynomial_of_isJacobson (hR : IsJacobson R) : IsJacobson R[X
 #align ideal.polynomial.is_jacobson_polynomial_of_is_jacobson Ideal.Polynomial.isJacobson_polynomial_of_isJacobson
 
 theorem isJacobson_polynomial_iff_isJacobson : IsJacobson R[X] ↔ IsJacobson R := by
-  refine' ⟨_, isJacobson_polynomial_of_isJacobson⟩
-  intro H
+  refine ⟨fun H => ?_, isJacobson_polynomial_of_isJacobson⟩
```

Comment:
```suggestion
  refine ⟨fun H ↦ ?_, isJacobson_polynomial_of_isJacobson⟩
```
Isn't this the preferred style in Mathlib?

Reply:
It's not my favorite thing. I just reverted it. It was kinda out of scope anyway.